packages/sdk/src/create/create.ts

```diff
@@ -10,7 +10,7 @@ import { createLibp2pAndUpdateOptions } from "./libp2p.js";
  * Uses Waku Filter V2 by default.
  */
 export async function createLightNode(
-  options: CreateWakuNodeOptions = {}
+  options: ProtocolCreateOptions = {}
```

Comment:
`ProtocolCreateOptions` used for a `createNode` argument -- perhaps change naming/functionality here?